### DISCUSSION TITLE
mac safari - translation does not work when logged into deepl pro account

### DISCUSSION THREAD

**Discussion by ntspaceowl** [kishikawakatsumi/WebTranslator#244] [Q&A]
<img width="381" alt="스크린샷 2024-05-24 오후 8 54 25" src="https://github.com/kishikawakatsumi/WebTranslator/assets/44715147/ed3b34ac-6471-44e6-add0-0635f011eca7">
I have installed both safari for mac and safari for ipad, full page translation works on ipad but on mac it gives me an error message right away. I am attaching a screenshot. Please let me use it on mac (for reference, I tried logging into the official extension in chrome at the same time and it works fine, so I don't think it's an issue with the deepl server).

**Comment by kishikawakatsumi** [comment]
What is the URL of the website where the problem occurs and how can I reproduce the problem?

**Comment by ntspaceowl** [comment]
<img width="667" alt="image" src="https://github.com/kishikawakatsumi/WebTranslator/assets/44715147/2cc93fc4-9308-48fe-89c3-5f83f233c5e7">

It happens the same on all sites. 
Just to be sure, I logged out of the deepl in chrome extension, logged out again in safari -> logged in and tried it and immediately got the same reaction as the above snapshot.
The extension I'm using specifically in safari is an ad-ad, but I've already tested it with ad-ads turned off, and for me it's 100% happening in all situations at the moment.

**Reply by yarotravel** [reply]
For me is not working as well at all sites. But I am using MacOS Sequoia

**Comment by kishikawakatsumi** [comment]
I could not reproduce the problem in my environment, so I can't investigate anymore. If you are asking me to fix the problem, please provide me with additional information, such as what error response you are getting back using packet capture if you can.

<img width="1728" alt="Screenshot 2024-05-25 at 3 01 39" src="https://github.com/kishikawakatsumi/WebTranslator/assets/40610/be9cd9a4-f14d-4fbf-88ff-d492da7e55a1">
